client/src/components/pages/FilterPage/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+import axios from 'axios';
+import { notification } from 'antd';
+
+import Cards from '../../common/Card';
+import Filter from '../../common/Filter';
+import './style.css';
+
+class FilterPage extends Component {
+  state = {
+    data: [],
+  };
+
+  componentDidMount = async () => {
+    const result = await axios.get('/api/v1/intial');
+    this.setState(prevState => ({ ...prevState, data: result.data.data }));
+  };
+
+  openNotificationWithIcon = error => {
+    notification.error({
+      message: 'somthing wrong !!',
```

Comment:
```suggestion
      message: 'something wrong !!',
```
just write it correctly ^_^